.cmakelintrc

```diff
@@ -0,0 +1 @@
+filter=-linelength,-readability/wonkycase
```

Comment:
I would highly recommend the use of consistent command case

Reply:
Unfortunately it objects to things like FetchContent_Declare which is taken directly from the official [docs](https://cmake.org/cmake/help/latest/module/FetchContent.html).
Maybe this could be changed in future but for now I'd prefer to have the settings match the style of the code as it is.